src/engine/sample.h

```diff
@@ -106,10 +107,9 @@ struct DivSample {
   DivSampleDepth depth;
   bool loop;
   // valid values are:
-  // - 0: No loop
-  // - 1: Forward loop
-  // - 2: Backward loop
-  // - 3: Pingpong loop
+  // - 0: Forward loop
```

Comment:
What is this? You can't break compatibility!

Make 0 "Unused" and 1 "Forward" (and so on)

Reply:
It's for correcting incorrect info.
Because DIV_SAMPLE_LOOP_FORWARD = 0, DIV_SAMPLE_LOOP_BACKWARD = 1, (...)

Reply:
Oh!